README.md

```diff
@@ -0,0 +1,2 @@
+nva-person-reference
+is a library that read and persist personal preferences for NVA users
```

Comment:
Are these private, or public preferences? What was annotated in my overview was: "The public preferences for how users' own data should be displayed in searches", but maybe this is entirely wrong?